index.html

```diff
@@ -195,6 +195,30 @@ <h2>About
         </ul>
       </div>
     </div>
+    <div class="contact-me">
+      <h2 class="heading">Contact me</h2>
+      <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+        I’d love to help with it!</p>
+
+        <form action="https://formspree.io/f/{form_id}" method="post">
+      
+          <input type="text" id="name" name="name" maxlength="30" placeholder="Lorem Ipsum" required>
+          <br><br>
+          
+          <input type="email" id="email" name="email" placeholder="loremipsum@gmail.com" required>
+          <br><br>
```

Comment:
- [optional] great job implementing the contact design. Please kindly make it responsive up till the desktop version to make it appealing in all screens sizes. 
![Screenshot (1135)_LI](https://github.com/Edgahkipkemoi/mobile/assets/65298048/0133b233-1c9b-4652-b7a7-8a7e026cee81)

Reply:
can you get to me I need a brief explanation for this please.